packages/request-node/src/request/confirmedTransactionStore.ts

```diff
@@ -0,0 +1,66 @@
+import { DataAccessTypes, LogTypes } from '@requestnetwork/types';
+import * as httpStatus from 'http-status-codes';
+
+import * as Keyv from 'keyv';
+
+/**
+ * Class for storing confirmed transaction information
+ * This class allow to store and serve the information when a transaction is confirmed
```

Comment:
This is not necessary, it's a duplicate from the previous line.

Reply:
I think we need more clarity, why do we need a class to store confirmed transaction ?
What `serve the information` does mean ?

Reply:
I rephrased it